verification/curator-service/api/src/model/source.ts

```diff
@@ -31,31 +31,46 @@ const fieldSchema = new mongoose.Schema({
     },
 });
 
-interface Parsing {
+interface RegexParsing {
     fields: Array<Field>;
 }
 
-const parsingSchema = new mongoose.Schema({
+const regexParsingSchema = new mongoose.Schema({
     fields: [fieldSchema],
 });
 
+interface Parser {
+    awsLambdaArn: string;
+}
+
+const parserSchema = new mongoose.Schema({
+    awsLambdaArn: {
+        type: String,
+        required: 'Enter a parser AWS Lambda ARN',
+    },
+});
+
+interface Schedule {
+    awsRuleArn: string;
+}
+
+const scheduleSchema = new mongoose.Schema({
```

Comment:
design also mentions awsScheduleExpression for the cron -- adding later?

Reply:
Yeah, was planning to. I'm waffling on this a little; on one hand, it's a property of the rule, so I don't like incurring the cost of synchronizing it in our collection. On the other hand, I think it'll make the read/update flow simpler. Either way, was punting in this PR.